# Message describing single TopoNavEdge

string edge_id
time last_updated
int64 start_node_id
int64 end_node_id
float64 cost
int64 type
